proposals/lip-0065.md

```diff
@@ -43,13 +43,13 @@ Events have the following properties:
 * `name`: A string indicating the name of the event.
 * `data`: A byte value. This property contains extra information regarding the event. Each module defines the schemas used for serialization and deserialization.
 * `topics`: An array of byte values. The event is inserted in the sparse Merkle tree once for each entry in this array.
-* `height`: The height of the block that the emitted the event. Together with the `index`, it uniquely identifies an event within a blockchain.
+* `height`: The height of the block in which the event was emitted. Together with the `index`, it uniquely identifies an event within a blockchain.
 * `index`: An integer counter (unique within a block) indicating how many events were emitted before the current one.
 
 ### Event Tree
 
 Events are emitted during [application domain stages of the block processing][lip-0055#block-processing-stages] and are inserted in a [sparse Merkle tree][lip-0039] as new leaf nodes. The leaf values are set to the serialized events. Leaf keys are given by the concatenation of a generic byte value and an index that identifies the position in which the event was emitted during the block processing. In particular, each event specifies an array of topics (similar to the 'indexed' keyword used in [Solidity](https://docs.soliditylang.org/en/v0.8.11/cheatsheet.html#modifiers)). The same event is inserted multiple times in the sparse Merkle tree, once for each entry in this array. By default, the indexed topics array contains the ID of the transaction that emitted the event or a constant placeholder for events emitted during the processing of the block header.
-Furthermore, events emitted during the [processing of a cross-chain message][lip-0053#ccm-processing] contain the ID of the cross-chain message as an extra default topic.
+However, events emitted during the [processing of a cross-chain message][lip-0053#ccu-processing] contain the ID of the cross-chain message as default topic rather than the ID of the cross-chain update transaction that contained the cross-chain message.
```

Comment:
```suggestion
However, events emitted during the [processing of a cross-chain message][lip-0053#ccu-processing] contain the ID of the cross-chain message as default topic instead of the ID of the cross-chain update transaction that contained the cross-chain message.
```

Reply:
Not a must, but sounds a bit vague otherwise